devices/profalux.js

```diff
@@ -0,0 +1,34 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const e = exposes.presets;
+const ea = exposes.access;
+
+module.exports = {
+    fingerprint: [{type: 'Router', manufacturerID: 4368, endpoints: [{ID: 1, profileID: 260, deviceID: 512},]}],
```

Comment:
Why not also add the input/output cluster like https://github.com/Koenkk/zigbee2mqtt/issues/11206#issuecomment-1035290270?